### PR TITLE
Update programmatic-filtering.md

### PR DESCRIPTION
**Pull request by martinivanoff**

### CODE REVIEW

**Review by kerpecheva — DISMISSED** [review]

File: controls/datagrid/filtering/programmatic-filtering.md [hunk 1]
```diff
@@ -129,21 +129,27 @@ It exposes the following properties:
 
 ### Nested Property Text Filter Descriptor
 
-The `NestedProprtyTextFilterDescriptor` is a descriptor which filters the nested properties.
+The `NestedPropertyTextFilterDescriptor` is a descriptor which allows you to filter the nested properties.
 
 It exposes the following properties:
 
 * `PropertyName`&mdash;Gets or sets the name of the property that is used to retrieve the filter value.
+* `ItemPropertyGetter`&mdash;Sets a custom function that should implement the access to the nested property. The function gets the value accessed by the `PropertyName` as the input argument and it should return the nested property as output result. If `ItemPropertyGetter` is not set, the filter descriptor behaves the same as [TextFilterDescriptor](#text-filter-descriptor).
```
Comment (yordan-mitev) [inline controls/datagrid/filtering/programmatic-filtering.md:137]:
```suggestion
* `ItemPropertyGetter`&mdash;Sets a custom function that implements the access to the nested property. The function gets the value accessed by the `PropertyName` as an input argument and returns the nested property as an output result. If `ItemPropertyGetter` is not set, the filter descriptor behaves the same as [TextFilterDescriptor](#text-filter-descriptor).
```

File: controls/datagrid/filtering/programmatic-filtering.md [hunk 2]
```diff
@@ -129,21 +129,27 @@ It exposes the following properties:
...
-                                           IsCaseSensitive="Falses"
-                                           Value="Barcelona"/>
+```C#
+var filterDescriptor = new NestedPropertyTextFilterDescriptor();
+filterDescriptor.PropertyName = "MyProperty";
+filterDescriptor.Operator = Telerik.Maui.Controls.Compatibility.Common.Data.TextOperator.EqualsTo;
+filterDescriptor.Value = "Expected value";
+filterDescriptor.ItemPropertyGetter = (originalPropertyValue) =>
+{
+    // The "originalPropertyValue" is the object fetched by the PropertyName of the descriptor. In this case value of the MyProperty property.
```
Comment (yordan-mitev) [inline controls/datagrid/filtering/programmatic-filtering.md:149]:
```suggestion
    // The 'originalPropertyValue' is the object fetched by the 'PropertyName' of the descriptor. In this case, this is the value of 'MyProperty'.
```

File: controls/datagrid/filtering/programmatic-filtering.md [hunk 3]
```diff
@@ -217,4 +223,4 @@ Add the `DelegateFilterDescriptor` to the `RadDataGrid` instance:
 dataGrid.FilterDescriptors.Add(new DelegateFilterDescriptor() { Filter = new CustomFilter()});
 ```
 
->important You could check our Programmatic Filtering example referring to the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}).
\ No newline at end of file
+>important You could check our Programmatic Filtering example referring to the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}).
```
Comment (yordan-mitev) [inline controls/datagrid/filtering/programmatic-filtering.md:226]: When referring to the SDKBrowser, always use the same sentence as a template. This makes the pages predictable and saves the developer time. You can adjust the sentence below so it fits this specific scenario:

"For a runnable example with the Button Getting Started scenario, see the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}) and go to Button > Getting Started."

Comment (kerpecheva) [reply controls/datagrid/filtering/programmatic-filtering.md]: Updated

**Review by yordan-mitev — APPROVED** [review]

**Pull request merged**